components/logrocket.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from "react";
+import { createService } from "./service";
+import LogRocket from "logrocket";
+import { useAuth } from "./auth";
+
+export const { Provider: LogRocketProvider } = createService(() => {
+    useEffect(() => {
+        const logrocketId = 'a9zjgu/maple'
```

Comment:
Let's move this constant outside the function, to the top-level of the module, and rename it `logRocketId`